aiidalab_qe/app/steps.py

```diff
@@ -1077,6 +1191,12 @@ def _extract_report_parameters(
         ] = builder.bands.bands_kpoints_distance.value
         parameters["nscf_kpoints_distance"] = builder.pdos.nscf.kpoints_distance.value
 
+        tot_charge = builder.relax.base["pw"]["parameters"]["SYSTEM"]["tot_charge"]
+        if tot_charge:
+            parameters["tot_charge"] = tot_charge
+        else:
+            parameters["tot_charge"] = 0.0
+
```

Comment:
```suggestion
        parameters["tot_charge"] = builder.relax.base["pw"]["parameters"]["SYSTEM"].get("tot_charge", 0.0)
```

And probably better having a `_DEFAULT_TOT_CHARGE` for the class.

Reply:
@unkcpz I  solve it but i didnt implement the _DEFAULT_TOT_CHARGE for the class, since that is a function